frontend/src/components/phones/onboarding/RealPhoneSetup.module.scss

```diff
@@ -274,3 +260,162 @@
     text-align: center;
   }
 }
+
+// 3rd party custom input for verifying true phone number
+/* stylelint-disable selector-class-pattern */
+.step :global {
+  .PhoneInput {
+    /* This is done to stretch the contents of this component. */
```

Comment:
@Vinnl this CSS comes from a 3rd party package we are using. We can't modify the CSS classes directly. Same issue we had with the Toast component. Let me know if you want to approach this a different way. We might want to revisit some of these variables.